Weather/Services/UserLocations/RealmUserLocationsService.swift

```diff
@@ -0,0 +1,85 @@
+import Foundation
+import RealmSwift
+
+
+class RealmWeatherLocation: Object {
+    dynamic var locationId: String = ""
+    dynamic var name: String = ""
+    dynamic var region: String = ""
+    dynamic var country: String = ""
+    var latitude = RealmOptional<Double>()
+    var longitude = RealmOptional<Double>()
+}
+
+
+class RealmUserLocationsService: UserLocationsService {
+    
+    private let realm: Realm?
+    
+    init() {
+        do {
+            self.realm = try Realm()
+        } catch let error {
+            self.realm = nil
+            print(error)
+        }
+    }
+    
+    // MARK: <UserLocationsService>
+    
+    func storeLocation(location: WeatherLocation) {
+        let realmLocation = self.mapLocation(location)
+        
```

Comment:
I'm not sure if this logic belongs to the interactor or to the service. What do you think?